src/lib.rs

```diff
@@ -76,6 +76,8 @@ pub struct Options<'a> {
     pub newlines_after_list: usize,
     pub newlines_after_blockquote: usize,
     pub newlines_after_rest: usize,
+    /// Token count for fenced code block. An appropriate value of this field can be decided by
```

Comment:
Now I see why it was called `token` count, not `backtick` count - `~` can be used instead of backticks.